mlu_op.h

```diff
@@ -14523,6 +14523,112 @@ mluOpLgamma(mluOpHandle_t handle,
             const mluOpTensorDescriptor_t y_desc,
             void *y);
 
+/*!
+ * @brief Computes the Cholesky factorization of a Hermitian positive-definite matrix.
+ *
+ * @param[in] handle
+ *   Handle to a Cambricon MLUOP context that is used to manage MLU devices and queues in the
+ *   deformable convolution operation. For detailed information, see ::mluOpHandle_t.
+ *
+ * @param[in] input_desc
+ *  The descriptor of the input matrix to factorise, it is an n×n Hermitian matrix,
+ *  only the lower or upper part is meaningful.
```

Comment:
*  The descriptor of the input matrix to factorize. It is an n×n Hermitian matrix,
 *  but only the lower or upper part is meaningful.